ui/appui-abstract/src/appui-abstract/utils/UiEventDispatcher.ts

```diff
@@ -41,7 +41,7 @@ export class UiEventDispatcher {
     this._secondaryTimeoutPeriod = this._timeoutPeriod / 2;
   }
 
-  /** @internal - used for testing only */
+  /** @public - used for testing only */
```

Comment:
we should avoid publishing something as public if it's only used for testing purposes.

Does there need to be a test in AppUI that covers this or can that test be in AppUI abstract?

Reply:
This is used by the unit tests for the SyncUiEventDispatcher class in appui-react.